unstructured/ingest/cli/cmds/salesforce.py

```diff
@@ -31,11 +32,11 @@ def get_cli_options() -> t.List[click.Option]:
                 help="For the Salesforce JWT auth. Found in Consumer Details.",
             ),
             click.Option(
-                ["--private-key-path"],
+                ["--private-key"],
                 required=True,
-                type=click.Path(file_okay=True, exists=True, dir_okay=False),
-                help="Path to the private key for the Salesforce JWT auth. "
-                "Usually named server.key.",
+                type=FileOrJson(allow_raw_str=True),
```

Comment:
Will the contents of the private key always be a valid json? If so, lets use the default `allow_raw_str` to raise an error if the value passed in is not a valid json.

Reply:
I have changed type to `str`. This parameter should hold path or PEM key. Full validation is done in `SalesforceAccessConfig`.